docs/tutorials/CAT_Launch_Process_Linux_MacOS.md

```diff
@@ -34,6 +35,21 @@ For any questions regarding this tutorial, head over to the #chialisp channel on
 
 -----
 
+## CAT issuance granularity
+
+CAT denominations, as well as the rules behind minting and melting, can take some getting used to. Here are a few things to keep in mind before you issue your CATs:
+
+* Most Chia wallets choose to display their value in XCH. However, this is a purely cosmetic choice because Chia's blockchain only knows about mojos. One XCH is equal to one trillion (1,000,000,000,000) mojos.
+* One CAT token is equal to 1000 XCH mojos.
```

Comment:
We may want to make it clear that this is a SHOULD standard and not a MUST in that deviating from that rule can be done but might hurt interoperability

Reply:
OK, I can clarify that 1000 XCH mojos should be a CAT but doesn't necessarily need to be one. However, I think I should emphasize that you should have a very good reason to change this amount because it could break interoperability.